src/types/pageURL.ts

```diff
@@ -19,3 +15,9 @@ export const ACCOUNTS_SIGNOUT_PATH = `${ACCOUNT_URL}/signout`;
 export const HEALTHCHECK = "/healthcheck";
 
 export const ACCESSIBILITY_STATEMENT = "/accessibility-statement";
+
+export const PERSONS_NAME = "/what-is-the-persons-name";
+
+export const PERSONAL_CODE = "/companies-house-personal-code";
+
+export const WHAT_IS_EMAIL_ADDRESS = "/what-is-their-email-address";
```

Comment:
to maintain consistency with persons name, can you change WHAT_IS_EMAIL_ADDRESS to EMAIL_ADDRESS please